http/metrics.go

```diff
@@ -31,6 +31,8 @@ func (opt *metricsOption) WrapHandler(handler http.Handler) (http.Handler, error
 		},
 		[]string{"code", "method"},
 	)
+	prometheus.Unregister(latency)
```

Comment:
is this going to happen on every request or just server config?

Reply:
Only on init